cmd/minikube/cmd/docker-env.go

```diff
@@ -326,7 +326,11 @@ docker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/doc
 			// so directly set --ssh-host --ssh-add to true, even user didn't specify them
 			sshAdd = true
 			sshHost = true
+		}
+
+		d := co.CP.Host.Driver
 
+		if sshHost {
```

Comment:
this should be an OR so we dont break the old behavior

Reply:
so the question is @afbjorklund would user ever need to use ssh-add without Docker using SSH-HOST ? 
so do we need to some reason keep the old behviour or ssh-add to be working regardless if the user is using SSH or HTTPS